spec/models/ingredient_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe Ingredient, type: :model do
+  describe "validations" do
+    it { should validate_presence_of :name }
+    it { should validate_presence_of :calories }
+  end
+
+  describe "relationships" do
+    it { should have_many :dish_ingredients }
+    it { should have_many(:dishes).through(:dish_ingredients) }
+  end
+end
```

Comment:
Great work with the unit tests as well! Nicely done all around! You mentioned that you got a bit in your head for this IC. Please take a look at all these comments together so that you can reflect on these truths: 1) you crushed this IC 2) you're doing very well with this inning's technical content 3) you've got this! 

Keep up the great work!